TSResourceFile.js

```diff
@@ -170,8 +171,6 @@ TSResourceFile.prototype.getContent = function() {
             if (resource.getSource() && resource.getTarget()) {
                 this.logger.trace("writing translation for " + resource.getKey() + " as " + resource.getTarget());
 
-                filename = this.getFileName(resource.getPath());
-
```

Comment:
filename is still used on line 177, so you should still define it

Reply:
The same code is on line 164. It's a duplicate code.

Reply:
Oh yeah, I missed that!